lib/src/bindings.dart

```diff
@@ -31,3 +31,15 @@ abstract class JsEvent {
   external String get resource;
   external dynamic get timestamp;
 }
+
+//TODO: Perhaps should be exposed in pkg/node_interop?
```

Comment:
Good idea. And I'm also trying to follow a convention of using `Js` prefix for all external objects, makes it easier to read code (kind of picked it up from `dart:js` with JsObject, JsArray, etc).